docs/app/pages/components/components-sections/wizard/wizard/wizard.component.ts

```diff
@@ -58,6 +58,14 @@ export class ComponentsWizardComponent extends BaseDocumentationSection implemen
         this._announcer.announce(`${this.steps[index].header} activated`);
     }
 
+    userValidator = function (): boolean {
```

Comment:
This should be:

```typescript
userValidator(): boolean {
    ...
}```

We should also remove this `something` code, either just return `true` or put some logic in here,